corehq/apps/hqmedia/view_helpers.py

```diff
@@ -114,18 +115,42 @@ def update_multimedia_paths(app, paths):
 
 def download_audio_translator_files(domain, app, lang):
     # Get bulk app translation single sheet data
+    headers = get_bulk_app_sheet_headers(app, single_sheet=True, lang=lang, eligible_for_transifex_only=True)
+    headers = headers[0]    # There's only one row since these are the headers for the single-sheet format
+    headers = headers[1]    # Drop the first element (sheet name), leaving the second (list of header names)
+    text_index = headers.index('default_' + lang)
+    audio_index = headers.index('audio_' + lang)
```

Comment:
how about
`audio_text_index` and `audio_path_index`?

Reply:
sure, b1693cd9b2b